runtime/altair/src/lib.rs

```diff
@@ -2164,9 +2165,9 @@ impl_runtime_apis! {
 
 
 	// Investment Runtime APIs
-	impl runtime_common::apis::InvestmentsApi<Block, AccountId, TrancheCurrency, CurrencyId, PoolId, Balance> for Runtime {
-		fn investment_portfolio(account_id: AccountId) -> Option<Vec<(PoolId, CurrencyId, TrancheCurrency, Balance)>> {
-			runtime_common::investment_portfolios::get_portfolios::<Runtime, AccountId, TrancheId, Investments, TrancheCurrency, CurrencyId, PoolId, Balance>(account_id)
+	impl runtime_common::apis::InvestmentsApi<Block, AccountId, TrancheCurrency, InvestmentPortfolio<Balance>> for Runtime {
+		fn investment_portfolio(account_id: AccountId) -> Vec<(TrancheCurrency, InvestmentPortfolio<Balance>)> {
+			runtime_common::investment_portfolios::get_account_portfolio::<Runtime, PoolSystem>(account_id)
```

Comment:
NOTE: Maybe we should bump the version to V2 but Apps has not used this API so far.

Reply:
Then I would just overwrite as you did.